UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Utils/AtmosConstants.cs

```diff
@@ -25,4 +25,18 @@ public static class AtmosConstants
 	public const float ONE_ATMOSPHERE = 101.325f;
 
 	public const float MINIMUM_HEAT_CAPACITY = 0.0003f;
+	public const float BARELY_COLD_HEAT = 260;
+	public const float ABIT_COLD_HEAT = 210;
+	public const float COLD_HEAT = 160;
+	public const float FREEZING_HEAT = 110;
+	public const float MELTING_HEAT = 460;
+	public const float HOT_HEAT = 410;
+	public const float ABIT_HOT_HEAT = 360;
+
+	public const float CELSIUS_BARELY_COLD_HEAT = 12f;
+	public const float CELSIUS_FREEZING_HEAT = 0f;
+	public const float CELSIUS_NORMAL_HEAT = 20;
+	public const float CELSIUS_WARM_HEAT = 28f;
+	public const float CELSIUS_HOT_HEAT = 38f;
+	public const float CELSIUS_MELTING_HEAT = 60f;
```

Comment:
These have been incorrectly converted.

Reply:
They're not incorrectly converted because the UI uses different numbers compared to the body (old and new health shinangins according to Bod)

Reply:
mind explaining a little bit more? So I get the full picture.

Reply:
I just realized something while I was writing an explanation for this.

If the UI uses old temperature numbers, why not just update the numbers to use the ones we use already for HealthV2?
Need to investigate this more

Reply:
that is what I was thinking indeed

Reply:
I misunderstood something bod said and apparently the numbers that exist already work correctly as is for now, also Dan's suggestion works, so yay.

review pls